src/components/UI/navbar/index.js

```diff
@@ -87,48 +93,48 @@ const Navbar = () => {
           </div>
-          <div
-            className={
-              mountedComponent
-                ? `${styles.navBarLogin}`
-                : `${styles.navBarLogin} d-none`
-            }
-          >
-            <Link href={LOGIN_URL}>
-              <a className={`${styles.btnLogin}${isLoggedIn ? 'd-none' : ''}`}>
-                <button type="button" className={styles.btnLoginText}>
-                  Sign In
-                  <img
-                    className={styles.githubLogo}
-                    src={GITHUB_LOGO}
-                    alt="GitHub Icon"
-                    height="15px"
-                    width="15px"
-                  />
-                </button>
-              </a>
-            </Link>
-            <div className={`${styles.userGreet}${isLoggedIn ? 'd-none' : ''}`}>
-              <Link href={USER_PROFILE_URL}>
-                <a>
-                  <div className={styles.userGreetMsg}>
-                    {isLoggedIn
-                      ? `Hello, ${userData.firstName}!`
-                      : `Hello, User!`}
-                  </div>
-                  <img
-                    className={styles.userProfilePic}
-                    src={
-                      isLoggedIn
-                        ? `${userData.profilePicture}`
-                        : `${DEFAULT_AVATAR}`
-                    }
-                    alt="Profile pic"
-                  />
-                </a>
-              </Link>
+          {mountedComponent && (
+            <div className={styles.navBarLogin}>
+              {!isLoggedIn && (
+                <Link href={LOGIN_URL}>
+                  <a className={styles.btnLogin}>
+                    <button type="button" className={styles.btnLoginText}>
+                      Sign In
+                      <img
+                        className={styles.githubLogo}
+                        src={GITHUB_LOGO}
+                        alt="GitHub Icon"
+                        height="15px"
+                        width="15px"
+                      />
+                    </button>
+                  </a>
+                </Link>
+              )}
+              {isLoggedIn && (
+                <div className={styles.userGreet}>
+                  <Link href={USER_PROFILE_URL}>
+                    <a>
+                      <div className={styles.userGreetMsg}>
+                        {`Hello ${
+                          isLoggedIn ? `${userData.firstName}` : 'User'
+                        }!`}
+                      </div>
+                      <img
+                        className={styles.userProfilePic}
+                        src={
+                          isLoggedIn
+                            ? `${userData.profilePicture}`
+                            : `${DEFAULT_AVATAR}`
+                        }
+                        alt="Profile pic"
+                      />
+                    </a>
+                  </Link>
+                </div>
```

Comment:
Could we go ahead with ternary operator?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

Reply:
NIT